src/test/java/com/example/neo4j/data/PersonData.java

```diff
@@ -0,0 +1,15 @@
+package com.example.neo4j.data;
+
+import com.example.neo4j.domain.Person;
+import reactor.core.publisher.Flux;
+
+public class PersonData {
+
+    public static Flux<Person> getPeople() {
```

Comment:
@amerkul test data should be visible in tests

Reply:
What exactly do you mean?

Reply:
@amerkul instead of calling `getPeople()` in test, create your people inside the test

Reply:
But I have to use this data into different test classes. I'm planning to create tests on service classes.